command/resource/apply/apply.go

```diff
@@ -171,23 +124,19 @@ Usage: consul resource apply [options] <resource>
 
 	$ consul resource apply -f=demo.hcl
 
-	Example (from file):
-
-	$ consul resource apply demo.hcl
-
 	Example (from stdin):
 
-	$ consul resource apply -
+	$ consul resource apply -f - < demo.hcl
 
 	Sample demo.hcl:
 
 	ID {
 		Type = gvk("group.version.kind")
 		Name = "resource-name"
 		Tenancy {
-		Namespace = "default"
-		Partition = "default"
-		PeerName = "local"
+			Partition = "default"
+			Namespace = "default"
+			PeerName = "local"
```

Comment:
You'll probably have to remove all references to `PeerName` before/during merge since it just got removed from the tenancy struct across the entire codebase.

Reply:
Thanks for letting me know! Double checked to make sure I don't have legacy `Peer` stuff in the resource cli command area